ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/response/ResponseMapper.java

```diff
@@ -71,7 +71,8 @@ public FullDefenceResponse from(CCDResponse response) {
             partyMapper.from(response.getDefendant()),
             statementOfTruth,
             FullDefenceResponse.DefenceType.valueOf(response.getResponseType().name()),
-            response.getDefence()
+            response.getDefence(),
+            null
```

Comment:
That is not right, you basically ignored new property in CCD world. Please implement storing the payment details in CCD as well.

Reply:
Will speak to Sabah about this as we decided to get data saved to claim store first

Reply:
Who decided it? At this point CCD data store gets the same level of support as database.